index.template.html

```diff
@@ -2335,6 +2334,74 @@ <h3><code>securityDefinitions</code> and <code>security</code></h3>
     ...
 }
 </pre>
+<p>
+    <a href="#oauth2securityscheme">OAuth2SecurityScheme</a> can specify different <code>flow</code>s, the next two examples report
+    the typical configuration of the <code>client</code> and the <code>device</code> flows. Typically, 
+    the <code>client</code> flow is used for machine to machine communication, consequently the first example below
+    shows a fictional Split Air Conditioner that can be controller by his companion thermostat using the 
+    pre-shared client id and client secret. On the other hand, the second example describe a <a>Thing Description</a>
+    of a device that offers the <code>device</code> flow for other Things that do not have a rich user 
+    interface. From the <a>Thing Description</a> prospective <code>device</code> flow is similar to <code>code</code>
+    but compliant clients will act differently following OAuth 2.0 specification. 
```

Comment:
This should say HOW these flows are different.  In particular technically both code and device require actual user input (in slightly different ways...) as part of the process, and IMO are probably best suited for onboarding processes.  Only "client" technically allows authentication to be delegated to a service without directly involving the user.  We are in the middle of updating the security best practices document to make this clearer but I expect that we will explicitly recommend that "client" be used for M2M purposes and "device" or "code" used only for onboarding or other cases where there is a display or browser involved.  IMO "client" is the important flow.  It might also be worthwhile to note (again) that implicit and password are deprecated (and not even in the TD spec, you have to use an extension).